pkg/api/rest/rest.go

```diff
@@ -61,8 +61,9 @@ type Lister interface {
 	// NewList returns an empty object that can be used with the List call.
 	// This object must be a pointer type for use with Codec.DecodeInto([]byte, runtime.Object)
 	NewList() runtime.Object
-	// List selects resources in the storage which match to the selector.
-	List(ctx api.Context, label labels.Selector, field fields.Selector) (runtime.Object, error)
+	// List selects resources in the storage which match to the selector. 'options' can be nil.
+	// TODO: Move 'label' and 'field' to 'options'.
+	List(ctx api.Context, label labels.Selector, field fields.Selector, options *api.ListOptions) (runtime.Object, error)
```

Comment:
I think we'll want to move to unversioned.ListOptions eventually, but this is fine now.